### PR TITLE
New handlers

### PR DESCRIPTION
**Pull request by 0blu**
- [X] [CMSG_GENERATE_RANDOM_CHARACTER_NAME](https://github.com/WowLegacyCore/HermesProxy/commit/68ebddf5e46f679becc1d63fb6282ca59073b7ca)
- [X] [CMSG_GET_ACCOUNT_CHARACTER_LIST](https://github.com/WowLegacyCore/HermesProxy/commit/3d1c6a9de29d6a90d721b6a6d239c13eaa75c394)
image

### CODE REVIEW

**Comment by 0blu** [comment]
Can someone verify the mail suggestions for TBC?

Example names were:
`Sømënämê`
`Süs`
`Süüß`
`Shorde`

File: HermesProxy/World/Server/Packets/AccountDataPackets.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        {
+            packet.WritePackedGuid128(AccountId);
+            packet.WritePackedGuid128(CharacterGuid);
+            packet.WriteUInt32(RealmVirtualAddress);
+            packet.WriteByteEnum(Race);
+            packet.WriteByteEnum(Class);
+            packet.WriteByteEnum(Sex);
+            packet.WriteUInt8(Level);
+
+            if (ModernVersion.AddedInVersion(ClientVersionBuild.V9_0_5_37503))
```
Comment (ratkosrb) [inline HermesProxy/World/Server/Packets/AccountDataPackets.cs:21]: Why are you checking retail build numbers? Those have no relation to classic.

Comment (0blu) [reply HermesProxy/World/Server/Packets/AccountDataPackets.cs:21]: What build should I use instead? `V1_13_7_37279`?

Comment (ratkosrb) [reply HermesProxy/World/Server/Packets/AccountDataPackets.cs:21]: So just remove the version check.

**Pull request merged**